app.ts

```diff
@@ -0,0 +1,312 @@
+                alert("Does not exists");
+                recordIdValue = "0";
+            } else if (typeof fromNumber === "number" && fromNumber >= 0) {
+                getRecords(fromNumber, toNumber);
+            }
+        });
+    }
+}
+
+function createHeadingGrid(headings: any) {
+    let headingColumns: HTMLElement | null = document.getElementById("column-headings-container"); // Headings
+    let headingsData = `<h1 class="column-heading">${headings}</h1>`;
+
+    (headingColumns as HTMLDivElement).innerHTML += headingsData;
+}
+
+function recordCount() {
+    return fetch("http://localhost:2050/recordCount", {
+        method: "GET",
+        headers: { "Content-Type": "application/json" },
+    })
+        .then(checkResponseError)
+        .then((response: Response) => response.text())
+        .then((data) => {
+            recordNumberTotal = JSON.parse(data);
+        })
+        .catch((error: Error) => {
+            console.log(error);
+        });
+}
+
+function headingRowCreation() {
+    return fetch("http://localhost:2050/columns", {
+        method: "GET",
+        headers: { "Content-Type": "application/json" },
+    })
+        .then(checkResponseError)
+        .then((response: Response) => response.text())
+        .then((data) => {
+            let headingDataList = JSON.parse(data);
+            let headings: string;
+
+            for (let i = 0; i < headingDataList.length; i++) {
+                headings = headingDataList[i];
+                createHeadingGrid(headings);
+            }
+            resizeScreenData();
+        })
+        .catch((error: Error) => {
+            console.log(error);
+        });
+}
+
+function dynamicGrid(columnData: any) {
+    let infoColumns: HTMLElement | null = document.getElementById("info-columns-container"); // Information
+    // Creates the row that the info will display and adds it to the infoColumnsArea.
+    let infoDataRow = `<div id="info-row-${columnData[0]}" class="info-rows"></div>`;
+
+    if (infoColumns !== null) {
+        infoColumns.innerHTML += infoDataRow;
+        // Gets the created rows.
+        let finalInfoDataRow = document.getElementById("info-row-" + columnData[0]);
+        if (finalInfoDataRow !== null) {
+            // Loops through
+            for (let x = 0; x < columnData.length; x++) {
+                let infoData = `<p class="info-row-data">${columnData[x]}</p>`;
+                finalInfoDataRow.innerHTML += infoData;
+            }
+        }
+    }
+}
+
+function resizeScreenData() {
+    let toNumber: number;
+    let recordNav: HTMLElement | null = document.getElementById("record-navigation-container");
+    let nextBtn: HTMLElement | null = document.getElementById("next-records-btn");
+    let previousBtn: HTMLElement | null = document.getElementById("previous-records-btn");
+    let navBtns = document.getElementById("navigation-btns");
+    if (recordNav !== null) {
```

Comment:
You should probably error or something if this happens, otherwise the user will not know whats wrong.